lib/miq_automation_engine/models/miq_ae_datastore/xml_yaml_converter.rb

```diff
@@ -2,7 +2,8 @@ module MiqAeDatastore
   class XmlYamlConverter
     def self.convert(xml, domain_name, export_options)
       temp_domain = MiqAeDatastore.temp_domain.downcase
-      MiqAeDomain.create!(:name => temp_domain, :priority => 100, :enabled => false)
+      MiqAeDomain.create!(:name   => temp_domain, :priority => 100, :enabled => false,
+                          :tenant => Tenant.root_tenant)
```

Comment:
Are you sure we want `Tenant.root_tenant` here?
I don't know that it should be something different, but it seems curious to me.